src/app/rundown-view/components/timeline-markers/timeline-markers.component.ts

```diff
@@ -40,6 +40,7 @@ export class TimelineMarkersComponent implements AfterViewInit, OnChanges {
 
   private readonly secondsPerSection: number = 5
   private readonly subsectionsPerSection: number = 5
+  private readonly timelineMarkerFillColor: string = '#5f6164'
```

Comment:
If we still want to use the css variables, then we can use the computed style of the canvas element like so:
```typescript
getComputedStyle(this.canvasElement.nativeElement).getPropertyValue('--dark-gray-color')
```